mopidy_spotify/library.py

```diff
@@ -63,20 +63,13 @@ def _lookup_album(self, sp_link):
     def _lookup_artist(self, sp_link):
         sp_artist = sp_link.as_artist()
         sp_artist_browser = sp_artist.browse(
-            type=spotify.ArtistBrowserType.NO_TRACKS)
+            type=spotify.ArtistBrowserType.FULL)
```

Comment:
That can't be right. sp_artist.browse (sp_artistbrowse_create) returns albums.

I thought it returned all albums when using FULL, but when I tried with Avicii now, it returned 100 albums, which seems to be a very round number, possibly indicating it has been cut off.

I tested with https://gist.github.com/trygveaa/6fc0244290d94223d540 (replacing input prompts with values).
NO_TRACKS: Took 1.5 seconds, returned 100 albums.
FULL: Took 7 seconds, returned 16 albums.

Also worth noting: The libspotify documentation says: "SP_ARTISTBROWSE_FULL: All information except tophit tracks. This mode is deprecated and will removed in a future release".